Telegram/SourceFiles/info/profile/info_profile_music_button.cpp

```diff
@@ -77,19 +77,30 @@ void MusicButton::paintEvent(QPaintEvent *e) {
 
 int MusicButton::resizeGetHeight(int newWidth) {
 	const auto padding = st::infoMusicButtonPadding;
-	const auto &font = st::infoMusicButtonTitle.style.font;
-
+	const auto& font = st::infoMusicButtonTitle.style.font;
 	const auto top = padding.top();
 	const auto skip = st::normalFont->spacew;
 	const auto available = newWidth - padding.left() - padding.right();
+	const auto minPerformer = std::min(80, available / 4);
```

Comment:
What happens when you change scale in Telegram settings?

Reply:
I believe it has to be
```suggestion
	const auto minPerformer = std::min(style::ConvertScale(80), available / 4);
```